### DISCUSSION TITLE
caching foreign files

### DISCUSSION THREAD

**Discussion by gregdenay** [bio-raum/FooDMe2#16] [General]
@marchoeppner 

hi, even with the `-resume` argument, the foreign files from SRA are staged everytime and all dependent jobs are rerun. 
Is it possible to cache the SRA files so that they are not downloaded everytime?

**Comment by marchoeppner** [comment]
Hi,

sadly, no. I think this has to do with fingerprinting the files. Nextflow cannot guarantee that the remote files haven't changed so it re-pulls them each time. 

I noticed that ENA is unavailable from time to time, so we could consider adding 2-3 libraries to the github repo as assets instead. But that only solves issue were ENA is down, not the pulling of remote files as such. You could set up a local config for a test with locally available files though, if you want to avoid downloading files over and over.
